wiki/Contests/KPMC/1/en.md

```diff
@@ -0,0 +1,86 @@
+---
+tags:
+  - KPMC1
+  - KPMC#1
+  - KPMC 1
+---
+
+# K-Pop Mapping Contest #1
+
+![KPMC#1 banner](img/banner.jpg)
+
+The **K-Pop Mapping Contest #1** (***KPMC#1***) is an osu!mania mapping contest which aimed to promote K-pop songs to a wider audience of mappers and players. It was the first instalment of the K-Pop Mapping Contest.
+
+## Contest schedule
+
+| Event | Timestamp (UTC) |
+| --: | :-- |
+| Submission phase | 2025-06-29 to 2025-08-17 |
+| Judging phase | 2025-08-18 to 2025-09-20 |
+| Results announcement | 2025-08-28 at 12 UTC |
```

Comment:
Same deal as the previous Date formatting changes. Datetimes also have their own format to follow, as seen [here](https://osu.ppy.sh/wiki/en/Article_styling_criteria/Writing#date-and-time-formatting)

```suggestion
| Results announcement | 2025-08-28 at (12:00 UTC) |
```